### ISSUE TITLE
Exception when wiping disk and error occurred

### ISSUE THREAD

**Issue by SylwesterZarebski** [TalAloni/HardDiskValidator#7]
I'm wiping old disk through USB-SATA interface and when error occured, all blocks are filled dark red (IO Error) and then program throws exception in message box. When clicked OK, program quits after few seconds.
Exception:
```
Exception: System.IO.IOException: Unable to set disk attributes, Error: 55 Source: HardDiskValidator    w DiskAccessLibrary.PhysicalDiskControl.TrySetOnlineStatus(SafeFileHandle hDevice, Boolean online, Boolean persist)
   w DiskAccessLibrary.PhysicalDiskControl.SetOnlineStatus(SafeFileHandle hDevice, Boolean online, Boolean persist)
   w DiskAccessLibrary.PhysicalDisk.SetOnlineStatus(Boolean online, Boolean persist)
   w HardDiskValidator.MainForm.PerformTest(Int32 physicalDiskIndex, TestName testName)
   w HardDiskValidator.MainForm.<>c__DisplayClassd.<btnStart_Click>b__8()
   w System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   w System.Threading.ThreadHelper.ThreadStart()
```
Image:
![image](https://user-images.githubusercontent.com/1755531/98014511-0521d700-1dfc-11eb-955d-b03012ee0d75.png)

PS. I know, i should throw away disk, but program should not crash after any error.

**Comment by TalAloni** [comment]
Hopefully this should be resolved in v1.1.1 - Thanks!

**Comment by SylwesterZarebski** [comment]
Thanks a lot! I'll try it next time i'll check suspect disk.
